addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.js

```diff
@@ -47,7 +47,8 @@ export class DiscussSidebarSubchannel extends Component {
 
 export class DiscussSidebarChannel extends Component {
     static template = "mail.DiscussSidebarChannel";
-    static props = ["thread"];
+    static props = ["thread", "isCategoryOpen?"];
+    static defaultProps = { isCategoryOpen: true };
```

Comment:
`isCategoryOpen` is not necessary. You can deduce from `thread.discussAsCategory.open`